app/Console/Commands/PhotosAddedNotification.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Mail\PhotosAdded;
+use App\Models\Configs;
+use App\Models\Photo;
+use App\Models\User;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Mail;
+
+class PhotosAddedNotification extends Command
+{
+	/**
+	 * The name and signature of the console command.
+	 *
+	 * @var string
+	 */
+	protected $signature = 'lychee:photos_added_notification';
+
+	/**
+	 * The console command description.
+	 *
+	 * @var string
+	 */
+	protected $description = 'Send email notifications for newly added photos';
+
+	/**
+	 * Create a new command instance.
+	 *
+	 * @return void
+	 */
+	public function __construct()
+	{
+		parent::__construct();
+	}
+
+	/**
+	 * Execute the console command.
+	 *
+	 * @return int
+	 */
+	public function handle()
+	{
+		$settings = Configs::get();
+
+		if ($settings['new_photos_notification']) {
+			$users = User::whereNotNull('email')->get();
+
+			foreach ($users as $user) {
+				$photos = [];
+
+				foreach ($user->unreadNotifications as $notification) {
+					$photo = Photo::find($notification->data['id']);
+
+					if ($photo && $photo->thumbUrl) {
+						if (!isset($photos[$photo->album_id])) {
+							$photos[$photo->album_id] = [
+								'name' => $photo->album->title,
+								'photos' => [],
+							];
+						}
+
+						$photos[$photo->album_id]['photos'][$photo->id] = [
+							'thumb' => config('app.url') . '/uploads/thumb/' . $photo->thumbUrl,
```

Comment:
Instead of using `config('app.url')` 
this should be using `Storage::url(...)`
There is no guarantee that the images are stored on the same server as Lychee in the future.
You may want to use a S3 bucket or something (when supported (hopefully)).

I would suggest to have a look at:
https://github.com/LycheeOrg/Lychee/blob/master/app/Models/Extensions/PhotoCast.php

Further more, thumbs are only square, you may consider to send the small version which preserve aspect ratio. :)